mkosi/distributions/fedora.py

```diff
@@ -172,10 +174,9 @@ def invoke_dnf(
         "--setopt=keepcache=1",
         "--setopt=install_weak_deps=0",
         f"--setopt=cachedir={state.cache_dir}",
-        f"--setopt=reposdir={' '.join(str(p) for p in state.config.repo_dirs)}",
```

Comment:
We drop this but never set it again to the path in the pkgmngr tree?

Reply:
That must have gotten lost during a rebase.